css/style.css

```diff
@@ -1,7 +1,355 @@
+/* ----------- VISUALIZATION ----------- */
+
 .circle {
   display: inline-block;
   width: 100px;
   height: 100px;
   border-radius: 50px;
   background-color: red;
 }
+
+/* ----------- UI - General ----------- */
+
+*::before, *::after, * {
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 170px auto;
+  height: 100vh;
+  margin: 0;
+  font-family: "Open Sans", sans-serif;
+  font-weight: normal; 
+}
+
+h1, h2, h3 {
+  margin: 0;
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 1.75em;
+}
+
+h2 {
+  font-size: .75em;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+h3 {
+  font-size: 1.2em;
+  color: #73A4B8;
+  text-transform: uppercase;
+}
+
+button {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+  margin-top: 10px;
+  text-align: left;
+  text-indent: calc(10px + 15px);
+  padding: 10px;
+  font-size: .75em;
+  border: none;
+  border-width: 1px;
+  border-radius: 3px;
+  cursor: pointer;
+  color: #EAEAEA;
+  box-shadow: 0 3px 3px #12181B;
+}
+
+.icon--button {
+  position: absolute;
+  top: 50%;
+  left: 10px;
+  transform: translateY(-50%);
+}
+
+.icon {
+  display: inline-block;
+  width: 15px;
+  height: 15px;
+  margin-right: 10px;
+  fill: #EAEAEA;
+}
+
+a {
+  color: #73A4B8;
+  font-size: .75em;
+}
+
+/* ----------- UI - Side Panel ----------- */
+
+aside {
+  background-color: #404850;
+  padding: 15px;
+}
+
+aside h1 {
+  color: #fff;
+}
+
+.logo {
+  max-width: 100%;
+}
+
+aside h2 {
+  margin-top: 20px;
+  color: #EAEAEA;
+}
+
+.nav__button {
```

Comment:
Where possible, I now use type selectors and more child selectors. The BEM class naming convention has been dropped.